test/extended/networking.sh

```diff
@@ -85,50 +99,81 @@ function save-artifacts() {
   copy-container-files "/etc/hosts" "${dest_dir}"
 }
 
-# Any non-zero exit code from any test run invoked by this script
-# should increment TEST_FAILURE so the total count of failed test runs
-# can be returned as the exit code.
-TEST_FAILURES=0
-function test-osdn-plugin() {
+function deploy-cluster() {
   local name=$1
   local plugin=$2
   local isolation=$3
+  local log_dir=$4
```

Comment:
I don't think it's required in this case - test-managed deployment with this method will always want to retain logs.